src/overlays/actors/ovl_En_Rr/z_en_rr.c

```diff
@@ -1,15 +1,75 @@
+/*
+ * File: z_en_rr.c
+ * Overlay: ovl_En_Rr
+ * Description: Like Like
+ */
+
 #include "z_en_rr.h"
+#include "vt.h"
 
 #define FLAGS 0x00000435
 
 #define THIS ((EnRr*)thisx)
 
+#define RR_MESSAGE_SHIELD (1 << 0)
+#define RR_MESSAGE_TUNIC (1 << 1)
+#define RR_MOUTH 4
+#define RR_BASE 0
+
+typedef enum { REACH_NONE, REACH_EXTEND, REACH_STOP, REACH_OPEN, REACH_GAPE, REACH_CLOSE } EnRrReachState;
+
```

Comment:
the formatter doing this to the enum is truly a shame lol. and trailing commas on enums induce warnings so, welp

Reply:
Can get it to stack properly by giving the enum a tag.